hls4ml/writer/vivado_writer.py

```diff
@@ -134,7 +134,7 @@ def _make_array_pragma(variable):
 
     @staticmethod
     def _make_stable_pragma(variable):
-        template = '#pragma HLS STABLE variable={name}'
+        template = '//#pragma HLS STABLE variable={name} // disabled'
```

Comment:
Why does this need to be disabled? It was found to be necessary to get the expected latency and II given the specified RF with `Strategy: Resource`

Reply:
@GiuseppeDiGuglielmo can comment more, but basically, we were seeing that with this pragma in older versions of Vivado (that he's using for deploying the models), cosim (RTL) was not replicating the csim results.

Reply:
With that pragma and Vivado 2019.1 the generated RTL is not functional correct. Tested first on a board and then in cosimulation.

Reply:
Okay interesting. I ran cosim at the time to check it didn't break things, and everything looked okay. It's something to look into further. Without this pragma, do you get the expected II for the layers given the RF?

Reply:
This is a portion of a larger model that we can use to observe the problem:
http://gandalf.cs.columbia.edu/projects/hls4ml/pynqz1_m_axi_8_serial_prj.tgz

There are only 3 _small_ layers:
```
dense ===> batch_norm ===> relu
      ^^^             ^^^
```

In Vivado 2019.1, if you run it (c-sim, hls, cosim) with the `STABLE` pragma for the two internal buffers (`^^^`), you will get `c-sim != cosim`. For example:
```
// c-sim
[0] 1.75 0 0 0 3.75 0 1.5 0 0.625 1.25 3.625 1.75 0 0 2.625 1.875 
// cosim
[0] 1.75 0 1.5 5.125 3.75 0.375 6.25 0.625 0.625 1.25 3.625 1.625 0 0 2.625 1.875 
```
If you comment the STABLE pragma away, then c-sim and cosim match.

STABLE seems to affect only the RELU behavior, but I am not sure if that is always true.

In terms of performance, with `STABLE` the II of this model is 1024 as expected (RF=1024). In addition:
```
lat 1039, bram 231, dsp 2, ff 15564, lut 19550
```
Without `STABLE` the II is the same, while we have some minimal estimated resource differences:
```
lat 1039, bram 231, dsp 2, ff 16779, lut 19858
```